src/RedisEnterpriseCache/RedisEnterpriseCache.Autorest/examples/Update-AzRedisEnterpriseCacheDatabaseDbRedisVersion.md

```diff
@@ -0,0 +1,22 @@
+### Example 1: {{ Add title here }}
```

Comment:
Please finish the examples and regenerated the module

Reply:
Added

Reply:
You should add the example here and the files of docs are all generated.

Reply:
Got it! updated now!